src/ConnectionConfig.tsx

```diff
@@ -72,21 +80,29 @@ export const ConnectionConfig: FC<ConnectionConfigProps> = (props: ConnectionCon
       <InlineField
         label="Authentication Provider"
         labelWidth={labelWidth}
-        tooltip="Specify which AWS credentials chain to use."
+        tooltip={`Specify which AWS credentials chain to use.${
+          tempCredsFeatureEnabled && awsAllowedAuthProviders.includes(AwsAuthType.GrafanaAssumeRole)
+            ? 'Note: Grafana Assume Role Authentication is only available to Grafana Cloud customers'
+            : ''
+        }`}
       >
         <Select
           aria-label="Authentication Provider"
           className="width-30"
           value={currentProvider}
-          options={awsAuthProviderOptions.filter((opt) => awsAllowedAuthProviders.includes(opt.value!))}
+          options={awsAuthProviderOptions.filter((opt) =>
```

Comment:
kind of nitpicky but it feels like you shouldn't have to change this, like instead focus on ensuring that awsAllowedAuthProviders only includes GrafanaAssumeRole if the feature is enabled and then you won't need any additional checks later on right?

Reply:
It was between adding a condition here, or a filter() for `settings.awsAllowedAuthProviders` [here](https://github.com/grafana/grafana-aws-sdk-react/blob/3b2a5097f173ed0e9cdc1a094009fd2fb48a3a69/src/ConnectionConfig.tsx#L41C1-L41C1) (since in theory, the allowedProviders could have this option in grafana settings, but the TempCreds feature flag disabled). I will add it up there instead if you think it's more readable though!